compliance/compliance.go

```diff
@@ -97,6 +97,9 @@ type Test struct {
 	// immediately NACK forward references, which causes some tests to fail. The reference
 	// implementation handles reodering.
 	RequiresServerReordering bool
+	// RequiresImplicitReplace marks a test that requires the implementation of AFTOperation
```

Comment:
nit: how about "RequiresImplicitReplace marks a test that ADD operations perform implicit replacement of existing entries"

Reply:
I know you're trying to shorten the comment, but the sentence structure "RequiresFoo marks a test that requires the implementation" as above makes more sense, even though it is more verbose.